src/ResourceMover/README.md

```diff
@@ -54,13 +54,13 @@ title: ResourceMover
 service-name: ResourceMover
 prefix: Az
 subject-prefix: $(service-name)
-branch: 7a56078
+branch: main
```

Comment:
Please use a commit instead of a branch name here.

Reply:
Sure